appeals/e2e/cypress/support/happyPathHelper.js

```diff
@@ -87,10 +89,40 @@ export const happyPathHelper = {
 		caseDetailsPage.clickManageAgreementToChangeDescriptionEvidence();
 		caseDetailsPage.clickLinkByText('View and edit');
 		caseDetailsPage.clickButtonByText('upload a new version');
-		caseDetailsPage.uploadSampleImg();
+		caseDetailsPage.uploadSampleFile(sampleFiles.img);
 		caseDetailsPage.clickButtonByText('Continue');
 		caseDetailsPage.clickButtonByText('Confirm');
 		caseDetailsPage.clickButtonByText('Confirm');
 		caseDetailsPage.validateBannerMessage('Document updated');
+	},
+
+	uploadDocsLpaq(caseRef) {
+		caseDetailsPage.clickReviewLpaq();
+		caseDetailsPage.clickAddNotifyingParties();
+		caseDetailsPage.uploadSampleFile(sampleFiles.document);
+		caseDetailsPage.clickButtonByText('Continue');
+		caseDetailsPage.clickButtonByText('Confirm');
+		caseDetailsPage.clickButtonByText('Confirm');
+		caseDetailsPage.validateBannerMessage('Document added');
+	},
+
+	uploadDocVersionLpaq(caseRef) {
+		caseDetailsPage.clickManageNotifyingParties();
+		caseDetailsPage.clickLinkByText('View and edit');
+		caseDetailsPage.clickButtonByText('Upload a new version');
+		caseDetailsPage.uploadSampleFile(sampleFiles.img);
```

Comment:
Just a thought to consider. Would `sampleFiles.image` be clearer as the name over `sampleFiles.img`? I'm not suggesting either way, just a thought